### PR TITLE
small tweaks

### PR DESCRIPTION
**Pull request by mattip**
A few random tweaks of the presentation. Only suggestions, feel free to reject all or some of the changes

xref gh-4

### CODE REVIEW

File: presentation.ipynb [hunk 1]
```diff
@@ -122,7 +122,8 @@
     " - `ndarray`: A generic, n-dimensional array data structure\n",
     " - Sophisticated machinery for operating on array data (vectorization, broadcasting, `ufuncs`)\n",
     " - Language extension/integration (C-API, `f2py`)\n",
-    "   * [Array API](https://docs.scipy.org/doc/numpy/reference/c-api.array.html) for accessing/extending array functionality in external libraries"
+    "   * [Array API](https://numpy.org/doc/1.17/reference/c-api.array.html) for accessing/extending array functionality in external libraries\n",
```
Comment (mattip) [inline presentation.ipynb:125]: I think numpy.org is now the canonical URL

File: presentation.ipynb [hunk 2]
```diff
@@ -122,7 +122,8 @@
...
+    " - Protocols for mimicing the very successful NumPy Python interfaces for other easey interoperability with other packages (more later)"
```
Comment (mattip) [inline presentation.ipynb:126]: Not sure if this needs a talking point here, feel free to remove

File: presentation.ipynb [hunk 3]
```diff
@@ -465,7 +467,9 @@
     "NumPy currently targets computation involving:\n",
     "\n",
     " * in-memory, homogenously-typed array data\n",
-    " * cpu-based"
+    " * cpu-based\n",
+    " \n",
+    "but provides protocols to allow developers of packages targeting other computational models hooks to seamlessly interoperate in Python."
```
Comment (mattip) [inline presentation.ipynb:472]: like above, not sure this is needed here

**Review by mattip — COMMENTED** [review]
The SIMD slide looks great

File: presentation.ipynb (hunk 2 above)
Comment (rossbar) [reply presentation.ipynb]: Thanks, nice catch

File: presentation.ipynb (hunk 1 above)
Comment (rossbar) [reply presentation.ipynb:125]: Thanks, nice catch

Comment (rossbar) [reply presentation.ipynb:125]: Thanks, nice catch

File: presentation.ipynb (hunk 3 above)
Comment (rossbar) [reply presentation.ipynb:472]: I think you're right that there should be some sort of explicit mention of the extensibility here, as it is an important feature (just maybe not one that *users* are very familiar with).

File: presentation.ipynb (hunk 2 above)
Comment (rossbar) [reply presentation.ipynb:126]: Thanks, nice catch

Comment (rossbar) [reply presentation.ipynb:126]: Regarding the added bullet-point about protocols, did you have the `__array_function__` and `__array_module__` protocols in mind, or something less specific. I just want to make sure I fully understand the suggestion

Comment (mattip) [reply presentation.ipynb:126]: I was trying to catch the essence of all the efforts we have made over the years. The latest are NEP 37, `__array_function__` and `__array_ufunc__`. There is also `__array__` and `__array_wrap__` for subclassing `ndarray`. Then  the [PEP 3118](https://www.python.org/dev/peps/pep-3118/) buffer protocol, by Travis Oliphant, that solves the interchange problem for non-object arrays and it turns out we have an earlier [Array Interface](https://numpy.org/devdocs/reference/arrays.interface.html). I might have missed one or two more.

**Comment by rossbar** [comment]
Thanks for the feedback @mattip , the protocol/interop info is very helpful - I will have to study up to make sure I have a more clear picture of everything that has been done in that arena. I think it's certainly relevant for this audience so once I am a bit more comfortable with the wording I will talking points. Thanks again!

**Comment by rossbar** [comment]
Suggestions incorporated in ab7898f. Thanks @mattip !

**Pull request closed without merging**